packages/kbn-router-to-openapispec/src/process_router.ts

```diff
@@ -26,6 +26,89 @@ import {
 import type { OperationIdCounter } from './operation_id_counter';
 import type { GenerateOpenApiDocumentOptionsFilters } from './generate_oas';
 
+interface PrivilegeGroupValue {
+  allRequired: string[];
+  anyRequired: string[];
+}
+
+interface PrivilegeGroups {
+  all: PrivilegeGroupValue;
+  serverless: PrivilegeGroupValue;
+  traditional: PrivilegeGroupValue;
+}
+
+const extractAuthzDescription = (route: InternalRouterRoute) => {
```

Comment:
[OAS with authz description] Automatically extracting description from privilege sets. Text is in free format now, just to showcase our intent.